src/Components/MailtoDropdown.js

```diff
@@ -0,0 +1,99 @@
+import preact from 'preact';
+import { Text, IntlProvider } from 'preact-i18n';
+import PropTypes from 'prop-types';
+import t, { t_r } from '../Utility/i18n';
+
+// TODO: I would like to extend this with country-specific services in the future. Unfortunately, I did not find a way
+// to produce links for GMX and Web.de.
+export const MAILTO_HANDLERS = {
+    mailto: {
+        link: d => `mailto:${d.email}?subject=${d.subject}&body=${d.body}`,
```

Comment:
While I was resolving the merge conflicts I realized that there is another, now useless method `RequestLetter::toMailtoLink()`. That should be removed if we decide to handle all this here.

Reply:
We indeed don't seem to use that anywhere else.